app/src/main/java/mollie/tictactoe/AndroidPlayerType.java

```diff
@@ -0,0 +1,8 @@
+package mollie.tictactoe;
+
+import ttt.PlayerType;
+
+public class AndroidPlayerType extends PlayerType {
+
+    public static final String HUMAN = "HUMAN";
```

Comment:
"Define humanity by humanity..." :-)
Should the variable be named human? It feels redundant...